Phaser/phaser-workshop.adoc

```diff
@@ -65,31 +65,35 @@ function spelOpbouwen() {
 
     // Onze speler heeft grond nodig om op te lopen. Dat doen we hier
     grond = this.physics.add.staticGroup();
-    grond.create(400, 568, 'platform').setScale(2).refreshBody();
+    // De grond is 2x zo groot als een normaal platform
```

Comment:
of 'grond' onnoemen naar 'terreinMaker/Toevoeger' ofzo, dan is het duidelijker dat je telkens weer een platform toevoegt aan het spel,

Reply:
Klinkt als een goed idee.